cmd/certificates/auth.go

```diff
@@ -170,8 +170,8 @@ func writeAuthInfo(ctx context.Context, authDB *certificates.AuthorizationDB, em
 	return nil
 }
 
-func writeTokenInfo(claimed, open certificates.Authorizations, w io.Writer) error {
-	groups := map[string]certificates.Authorizations{
+func writeTokenInfo(claimed, open authorizations.Authorizations, w io.Writer) error {
+	groups := map[string]authorizations.Authorizations{
```

Comment:
`authorizations.Authorizations` look a little bit weird :)

Reply:
What do you suggest?

Reply:
thats main problem, no idea at the moment.